msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Designing your meeting extension
 author: heath-hamilton
-description: Learn how to design apps in Teams meetings and get the Microsoft Teams UI Kit, in-meeting tab, use cases, responsive behavior, shared meeting stage, theme, and navigation.
+description: Learn how to design apps in Teams meetings and get the Microsoft Teams UI Kit, in-meeting tab and use cases, responsive behavior and shared meeting stage, and theme and navigation.
```

Comment:
Both version (previous and current edit) don't make sense. Please rephrase. See if reducing the clutter from this description helps. That is, instead of as many generic keywords, write what specifically does the article convey.

@v-habibaumme , @v-psambrani

Reply:
Resolved